src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -280,17 +270,17 @@ void generatePrecinctSummaryFiles(
       Map<String, Integer> numBallotsByPrecinct)
       throws IOException {
     Set<String> filenames = new HashSet<>();
-    for (String precinct : precinctRoundTallies.keySet()) {
-      String precinctFileString = getPrecinctFileString(precinct, filenames);
+    for (var entry : precinctRoundTallies.entrySet()) {
```

Comment:
I'm learning a lot in this PR! I didn't know about `var` either. Seems like a nice way to reduce the verbosity a bit. But it's not clear to me why you're using it here but not in some other places, e.g. a bit below here in `updateCandidateNamesInTally`.

Reply:
I didn't know about `var` either! I only went back and changed this to `var` in those places where naming the full type produced a line that was too long.

Reply:
This was introduced in JDK 10, but we never got around to using it much. One downside is that it can decrease readability, but I think tamird@'s use is good. I'd probably lean towards consistently doing it throughout each class it's introduced though (e.g. here and in `Tabulator`).

Reply:
Done.